transaction/core/src/tx.rs

```diff
@@ -271,14 +271,14 @@ impl TxOut {
         recipient: &PublicAddress,
         tx_private_key: &RistrettoPrivate,
         hint: EncryptedFogHint,
-        rng: &mut RNG,
+        _rng: &mut RNG,
     ) -> Result<Self, AmountError> {
```

Comment:
Can we remove this rng from the API?

Reply:
Good catch. I forgot about that. Will do.